<ui version="4.0" >
 <class>ParamPropertiesDialog</class>
 <widget class="QDialog" name="ParamPropertiesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameter Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Description:</string>
       </property>
       <property name="buddy" >
        <cstring>descriptionEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="nameEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QCheckBox" name="limitRangeCheckBox" >
       <property name="text" >
        <string>&amp;Limit Range:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="minSpinBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>3</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum" >
          <double>1000</double>
         </property>
         <property name="minimum" >
          <double>-1000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxSpinBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>3</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum" >
          <double>10000</double>
         </property>
         <property name="minimum" >
          <double>-1000</double>
         </property>
         <property name="value" >
          <double>1</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="descriptionEdit" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>37</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>limitRangeCheckBox</tabstop>
  <tabstop>minSpinBox</tabstop>
  <tabstop>maxSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ParamPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ParamPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>309</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel" >
     <x>335</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limitRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limitRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>303</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
